config/api/src/main/java/org/jboss/windup/config/Variables.java

```diff
@@ -199,21 +201,24 @@ public void removeVariable(String name)
      * Searches the variables layers, top to bottom, for the iterable having all of it's items of the given type. Return
      * null if not found.
      */
-    public Iterable<? extends WindupVertexFrame> findVariableOfType(Class<?> type)
+    public <T> Iterable<T> findVariableOfType(Class<T> type)
```

Comment:
This makes it technically possible to break usage of this method. (Calling it with a class that is not a Frame). Is that what you really wanted to do?